include/verilated_trace_defs.h

```diff
@@ -0,0 +1,39 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//=============================================================================
+//
+// Code available from: https://verilator.org
+//
+// Copyright 2001-2021 by Wilson Snyder. This program is free software; you
+// can redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//=============================================================================
+///
+/// \file
+/// \brief Verilated tracing types
+///
+/// This file is not part of the Verilated public-facing API.
+/// It is only for internal use by Verilated tracing routines.
+///
+//=============================================================================
+#ifndef VERILATOR_VERILATED_TRACE_DEFS_H_
+#define VERILATOR_VERILATED_TRACE_DEFS_H_
+
+enum vltscope_t {
+    VLT_SCOPE_MODULE = 0,
```

Comment:
```suggestion
enum VltTraceScope : char {
    VLT_TRACE_SCOPE_MODULE = 0,
    ...
```
Calling it just a "scope" is a bit confusing as there's many uses of the term scope.

Also please add comments before this saying that they must match the gtkwave/fstapi.h <whatever> enum values.